content/authorization.md

```diff
@@ -869,35 +888,44 @@ support the stand-alone "openid" workflow for patients and/or
 their authorized representatives.
 
 When retrieving an access token utilizing using an
-online_access refresh, the most likely cause of failures
-is that access has been revoked.  The following steps are
-recommended for the user experience:
+offline_access refresh, the most likely cause of failures
+is that access has been suspended or completely revoked.
+The following steps are recommended for the user experience:
 
 - Indicate that the application's access may have been
-  revoked.
+  suspended or revoked.
 - Offer a "more information" link/button, hyperlinked to
   the value returned in the parameter "error_uri".
 - Offer the ability for the user to re-request
   authorization for your client application.
 
-#### Considerations for Handling 'online_access' ####
-
-When retrieving an access token utilizing using an
-online_access refresh, the most likely cause of failures
-is that the user's session has been terminated.  The
-following steps are recommended for the user
-experience:
-
-- Indicate that the user may have logged out, or that
-  the server otherwise requires that they log in again.
-- If your application was invoked via a "launch", and
-  the persona being serviced is a provider, direct the
-  user to launch the application again.
-- Offer the user the opportunity to "log in again"
-  (initiate the authorization grant workflow again) as
-  appropriate.
-- Offer a "more information" link/button, hyperlinked to
-  the value returned in the parameter "error_uri".
+_NOTE_:  The authorization server will not explicitly indicate
+whether a token was revoked or suspended.  As a result, there
+are additional recommendations to improve the overall
+interaction with the end-user.
+
+The error_uri used in the hyperlink/button should be launched
+in a separate browser window/tab.  This is recommended
+because there is no callback/redirect mechanism to get the
+user back into the application once they take an action.
+
+Additionally, the error_uri may not provide an opportunity
```

Comment:
consistency - some places `error_uri` is `"error_uri"`.

Reply:
I removed the quotes and formatted with _code_ style font.